src/app/util/config.h

```diff
@@ -300,4 +300,5 @@
 #define EMBER_AF_CUSTOM_NETWORK_INIT_OPTIONS (EMBER_NETWORK_INIT_PARENT_INFO_IN_TOKEN | Z3_NETWORK_INIT_BEHAVIOR)
 #endif // EMBER_AF_CUSTOM_NETWORK_INIT_OPTIONS
 
+#define MILLISECOND_TICKS_PER_SECOND 1000
```

Comment:
Not sure whether this should go here or in `types_stub.h`.  And we should have an issue tracking auditing consumers of this one too.

Reply:
Moved to `types_stub.h`